### PR TITLE
Add support for adding and removing devices to and from collections

### PR DESCRIPTION
**Pull request by pavelnkaz**
Please review pull request ("Ready for Review")

### CODE REVIEW

File: m2x/v2/collections.py [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
 # https://m2x.att.com/developer/documentation/v2/collections
 class Collection(Resource, Metadata):
     COLLECTION_PATH = 'collections'
     ITEM_PATH = 'collections/{id}'
     ITEMS_KEY = 'collections'
+
+    def devices(self):
+        return CollectionDevice.list(self.api, collection_id=self.id)
+
+    def add_device(self, serial):
```
Comment (fsaravia) [inline m2x/v2/collections.py]: Why naming the second parameter `serial` this endpoint expects a `device_id` which is different than the serial number. I'd suggest we rename it to `device_id`

File: m2x/v2/collections.py [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+        path = self.subpath('/devices/{device_id}'.format(device_id=serial))
+        return self.api.put(path)
+
+    def remove_device(self, serial):
```
Comment (fsaravia) [inline m2x/v2/collections.py]: Same as above.

File: m2x/v2/collections.py (hunk 1 above)
Comment (pavelnkaz) [inline m2x/v2/collections.py]: For some reason, I had equivocated serial with device_id. I'll update the code to use device_id.

File: m2x/v2/collections.py [hunk 3]
```diff
@@ -1,9 +1,26 @@
-from m2x.v2.resource import Resource
```
Comment (fsaravia) [inline m2x/v2/collections.py]: Just for the sake of tidiness, I'd add the `Device` import without changing this line (that's it: keeping the `Resource` import where it is).

Comment (pavelnkaz) [inline m2x/v2/collections.py]: Sounds good. I pushed the change to keep the import for Resource as-is.

**Comment by fsaravia** [comment]
@pavelnkaz Thank you!

**Pull request merged**